src/platform/plugins/shared/dashboard/public/services/dashboard_backup_service.ts

```diff
@@ -30,7 +31,7 @@ const getPanelsGetError = (message: string) =>
     values: { message },
   });
 
-export type DashboardBackupState = Partial<DashboardState> & {
+export type DashboardBackupState = Partial<Writable<DashboardState>> & {
```

Comment:
I see a lot of additions of the `Writable` utility. I understand this is because the schema types are generated as `readonly`. As a nit, I wonder if it would be beneficial to export the `DashboardState` type as `Writable` rather than importing it with that type everywhere it's used.